gradle/generation/local-settings.gradle

```diff
@@ -45,83 +45,4 @@ configure(rootProject) {
       }
     }
   }
-
```

Comment:
Please keep the "deprecated" localSettings task fora while as all jenkins/github jobs execute them. We can remove it later, but for now add a "fake" task for compatibility.

Reply:
@colvinco I think this is still outstanding